recipe/meta.yaml

```diff
@@ -30,11 +32,13 @@ requirements:
     - libtool
     - pkg-config
   host:
+    - {{ mpi }}  # [mpi != "nompi"]
     - blosc
     - bzip2
     - hdf5
```

Comment:
If your mpi builds require hdf5 to have been built with mpi, you will need to explicitly depend on the right mpi variant of hdf5. See [h5py](https://github.com/conda-forge/h5py-feedstock/pull/45/files#diff-e178b687b10a71a3348107ae3154e44c) for an example.